### PR TITLE
zh create trans poseidon-firmament-alternate-scheduler.md

### PR DESCRIPTION
**Pull request by jiazxjason**
zh create trans poseidon-firmament-alternate-scheduler.md

### CODE REVIEW

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+| title                        | content_type | weight |
+| ---------------------------- | ------------ | ------ |
+| Poseidon-Firmament Scheduler | concept      | 80     |
```
Comment (sftim) [inline content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:3]:
```suggestion
---
title: Poseidon-Firmament 调度程序是
content_type: concept
weight: 80
---

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+| High Availability                                            | Y                            | N                            |                                                              |
+| Real-time metrics based scheduling                           | N                            | Y†                           | **†** Partially supported in Poseidon-Firmament using Heapster (now deprecated) for placing Pods using actual cluster utilization statistics rather than reservations. |
+| Support for Max-Pod per node                                 | Y                            | Y                            | Poseidon-Firmament scheduler seamlessly co-exists with Kubernetes default scheduler. |
+| Support for Ephemeral Storage, in addition to CPU/Memory     | Y                            | Y                            |                                                              |
+| {{< /table >}}                                               |                              |                              |                                                              |
+
+-->
+
+## 功能比较 {#feature-comparison-matrix}
```
Comment (sftim) [inline content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:154]: Also valid:
```suggestion
## 功能比较
```

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+{{< table caption="Feature comparison of Kubernetes and Poseidon-Firmament schedulers." >}}
```
Comment (sftim) [inline content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:156]: This caption has not been localized.

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+[Poseidon-Firmament 安装指引](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/install/README.md#Installation) 介绍了如何将 Poseidon-Firmament 部署到您的群集。
+
+## 性能对比
+
+{{< note >}} 有关 Poseidon-Firmament 调度程序和 Kubernetes 默认调度程序之间的详细吞吐量性能比较测试结果，请参考最新的基准测试结果。{{< /note >}}
+
+每个 Pod 的调度程序（例如 Kubernetes 默认调度程序）以小批量（通常一次）处理 Pod。这些调度程序具有以下关键缺陷：
+
+1. 调度程序尽早提交 pod 安排，并限制其他等待安排的 pods 选择。
+2. 在各个 pods 中摊销工作的机会有限，因为考虑将其单独安排。
```
Comment (sftim) [inline content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:214]:
```suggestion
1. 调度程序尽早提交 pod 安排，并限制其他等待安排的 pod 选择。
2. 在各个 pod 中摊销工作的机会有限，因为考虑将其单独安排。
```

**Comment by tengqm** [comment]
The status of the [Poseidon](https://github.com/kubernetes-sigs/poseidon/commits/master) project is a bit confusing.
It smells to me like being abandoned.

- No PRs coming in since Dec 2019
- PRs related to code changes date back to April 2019.

I'd suggest we stop maintaining this by removing it from English site as well.

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:3]: `Poseidon-Firmament 调度程序` :)

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md (hunk 3 above)
Comment (chenrui333) [reply content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:156]: good point.

Comment (chenrui333) [inline content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:156]:
```suggestion
{{< table caption="Kubernetes 默认调度程序和 Poseidon-Firmament 调度程序间的比较" >}}
```

File: content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+| Gang 调度                                                | N                       | Y                           |                                                              |
+| 支持预绑定持久卷的调度                                   | Y                       | Y                           |                                                              |
+| 支持本地卷和动态持久卷绑定调度                           | Y                       | N                           |                                                              |
+| 高可用性                                                 | Y                       | N                           |                                                              |
+| 基于实时指标的调度                                       | N                       | Y†                          | **†** 使用 Heapster（现已弃用）在 Poseidon-Firmament 中得到部分支持，以便使用实际群集利用率统计信息而不是保留来放置 Pod。 |
+| 每个节点支持Max-Pod                                      | Y                       | Y                           | Poseidon-Firmament 调度程序与 Kubernetes 默认调度程序无缝共存。 |
```
Comment (chenrui333) [inline content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:173]:
```suggestion
| 每个节点支持 Max-Pod                                      | Y                       | Y                           | Poseidon-Firmament 调度程序与 Kubernetes 默认调度程序无缝共存。 |
```

Comment (chenrui333) [reply content/zh/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md:173]: style issue

**Comment by chenrui333** [comment]
@tengqm, are we still going for a merge of this PR (considering we have removed the English content).

This is pretty well written translation though. :)

Thanks @jiazxjason!!

**Comment by chenrui333** [comment]
The better idea for future translation might be raising an issue first and tag tengqm for review before the actual translation work. :)

**Comment by chenrui333** [comment]
closing the PR for now.
/close

**Pull request closed without merging**